src/autoload/global.gd

```diff
@@ -80,11 +106,85 @@ func _ready() -> void:
 #		defense_cards[defense_card.rank].append(defense_card)
 
 
+func save_config() -> void:
+	var config_file = FileAccess.open(CONFIG_PATH, FileAccess.WRITE)
```

Comment:
`:=`🤓

Reply:
you are not without sin either my brother

Reply:
Me making this comment
![image](https://github.com/kdreese/gmtk-2023/assets/7307599/54ddfd7b-a162-4d04-aa4c-71c7ea81a596)